src/schema/__tests__/ecommerce/create_order_with_artwork_mutation.test.js

```diff
@@ -11,54 +12,62 @@ describe("Create Order Mutation", () => {
     const resolvers = {
       Mutation: {
         createOrderWithArtwork: () => ({
-          order: exchangeOrderJSON,
-          errors: [],
+          orderOrError: { order: exchangeOrderJSON },
         }),
       },
+      OrderOrFailureUnion: {
+        __resolveType(obj, context, info) {
+          if (obj.order) {
+            return "OrderWithMutationSuccess"
+          } else if (obj.error) {
+            return "OrderWithMutationFailure"
+          }
+        },
+      },
     }
 
     rootValue = mockxchange(resolvers)
   })
+
   it("creates order and returns it", () => {
-    const mutation = `
+    const mutation = gql`
```

Comment:
`gql` is a no-op template string function, but it tricks prettier/vscode into formatting our graphqls correctly